src/pydap/wsgi/app.py

```diff
@@ -245,12 +248,32 @@ def init(directory):
         shutil.copy(path, directory)
 
 
+class PyDapApplication(WSGIApplication):
+    """An application interface for configuring and loading
```

Comment:
Is this the interface class for loading a WSGI application into the WSGI framework?